aws-amplify/amplify-js
# feat(amazon-cognito-identity-js): Clears "__uploadInProgress" from local storage

#### Description of changes
Clears the `__uploadInProgress` key from local storage after the user logs out or the session expires.

`__uploadInProgress` will store metadata on s3 uploads that are resumable.

#### Issue #, if available
[#61](https://github.com/aws-amplify/amplify-js/issues/61)

#### Description of how you validated changes

Manual testing:
1. In sample app, delete the access token in local storage.
2. Delete/alter the refresh token in local storage
3. Refresh the page
4. User should be signed out and relevant user data in local storage should be cleared



#### Checklist



By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
